Libraries/LLM/Load.swift

```diff
@@ -26,7 +26,7 @@ public func load(
         case .id(let id):
             // download the model weights and config
             let repo = Hub.Repo(id: id)
-            let modelFiles = ["config.json", "*.safetensors"]
+            let modelFiles = ["*.safetensors"]
```

Comment:
Does this download the config.json if it is missing?  I wonder if we need two calls to snapshot?

Reply:
I just tested it, and it works. In my own app, I download the tokenizer files and then the model files separately.